app/controllers/application_controller/ci_processing.rb

```diff
@@ -1868,6 +1868,7 @@ def process_cloud_object_storage_buttons(pressed)
     task = pressed.sub("#{klass.name.underscore.to_sym}_", "")
 
     return tag(klass) if task == "tag"
+    return javascript_redirect(:controller => klass.name.underscore.to_sym, :action => "new") if task == "new"
 
```

Comment:
Should the button be visible both from `/cloud_object_store_container/show_list` and `/ems_storage/ID?display=cloud_object_store_containers`, it would probably make more sense to have this logic in ci_processing (which is included by both controllers).

@AparnaKarve would you prefer me to copy line 1871 to [above this line](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ems_common.rb#L366) and to [above this line](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/cloud_object_store_container_controller.rb#L20)?